examples/vic-gov-au/pages/test-complex-listings.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <!-- This is just for POC, do not merge this -->
+   <rpl-page-layout class="app-main" :sidebar="sidebar">
+
+    <template slot="breadcrumbs" v-if="breadcrumbs">
+      <rpl-breadcrumbs :crumbs="breadcrumbs" />
+    </template>
+
+    <template slot="aboveContent" >
+      <rpl-search-form
+        @search="getSearchResults(searchForm.filterForm.model.title || '')"
+        class="rpl-site-constrain--on-all"
+        v-bind="searchForm"
+        :title="title"
+        :subtitle="subtitle"
+      />
+      <rpl-divider />
+    </template>
+    <transition name="fade">
+      <rpl-search-results
+        :searchResults="searchResults"
+        :pager="searchResults.length === 0 ? undefined : pager"
+        :responseSize="searchResults.length === 0 ? 0 : searchOptions.responseSize"
+        :count="searchResults.length === 0 ? 0 : count"
+        :errorMsg="errorMsg"
+        :noResultsMsg="noResultsCopy"
+        :perPage="20"
+        @pager-change="changed"
+        v-if="!loading"
+      >
+        <template v-slot:results="resultsProps">
+          <rpl-col cols="full">
+            <rpl-search-results-table v-if="columns" ref="search-results-table" :items="resultsProps.searchResults" :columnConfig="columns" />
+          </rpl-col>
```

Comment:
I think it should work with other result listing like cards by putting a for loop with some other logic here? Are you able to provide some storybook test to demo it? We need to prove it in POC that this V2 works for existing features.

Reply:
Added more stories including card example